pkg/cmd/workspacemode/workspace_mode.go

```diff
@@ -31,7 +31,7 @@ var workspaceModeRootCmd = &cobra.Command{
 
 func Execute() {
 	cmd.SetupRootCommand(workspaceModeRootCmd)
-
+	workspaceModeRootCmd.AddGroup(&cobra.Group{ID: "workspace", Title: "Project & Workspace"})
```

Comment:
![image](https://github.com/user-attachments/assets/bf14ae68-bfa6-47a4-949e-9880210a1400)

I see that you added the "workspace mode" to have the separation as well. In that case, we would need the forward, info, start and stop commands to all be under the subtitle ( the cmds are located in pkg/cmd/workspacemode ).

The ID "workspace" should also be util.WORKSPACE_GROUP